lab_13/src/lib.c

```diff
@@ -0,0 +1,217 @@
+#include "lib.h"
+
+node_t *create_node(vehicle_t data)
+{
+    node_t *node = (node_t *)malloc(sizeof(node_t));
```

Comment:
I think you need to add a memory allocation check.

Reply:
resolved